packages/eslint-config-sentry-app/index.js

```diff
@@ -72,3 +74,24 @@ module.exports = {
     'sentry/no-digits-in-tn': ['error'],
   },
 };
+
+//reuses the common rule import/order from the package 'eslint-config-sentry-react'
+config.rules['import/order'] = eslintConfigSentryReact.rules['import/order']
+
+// and includes a sentry/getSentry package-specific rule - pathGroups
+config.rules['import/order'][1] = {
+  ...config.rules['import/order'][1],
+  pathGroups: [
+    {
+      pattern: '@emotion/styled',
+      group: 'external',
+    },
+    {
+      pattern: 'sentry*/**',
+      group: 'internal',
+    },
+  ],
+  pathGroupsExcludedImportTypes: ['builtin'],
+}
```

Comment:
This is a bit hard to read, since `groups` is the configuration we most care about for consistency's sake, what if we just extracted that so that the rule looks more familiar.

```suggestion
const config = {
...
rules: {
...
  'import/order': ['error', {
    groups: eslintConfigSentryReact.rules['import/order'][1].groups,
  pathGroups: [
    {
      pattern: '@emotion/styled',
      group: 'external',
    },
    {
      pattern: 'sentry*/**',
      group: 'internal',
    },
  ],
  pathGroupsExcludedImportTypes: ['builtin'],
}
...
}
```